web/src/components/tasks/LicenseStatusReceipt.tsx

```diff
@@ -77,48 +73,47 @@ const LicenseStatusLookup: Record<LicenseStatus, string> = {
   REVOKED: Config.licenseSearchTask.revokedStatusText,
 };
 
+const pendingStatuses: Set<LicenseStatus> = new Set([
+  "PENDING",
+  "DRAFT",
+  "SUBMITTED",
+  "UNDER_INTERNAL_REVIEW",
+  "SPECIAL_REVIEW",
+  "PENDING_DEFICIENCIES",
+  "DEFICIENCIES_SUBMITTED",
+  "CHECKLIST_COMPLETED",
+  "APPROVED",
+  "PENDING_RENEWAL",
+  "PENDING_REINSTATEMENT",
+]);
+
+const isPending = (licenseStatus: LicenseStatus): boolean => {
+  return pendingStatuses.has(licenseStatus);
+};
+
 export const LicenseStatusReceipt = (props: Props): ReactElement => {
-  const [theme, setTheme] = useState<PermitTheme>(pendingPermitTheme);
+  let theme = pendingPermitTheme;
```

Comment:
pebble - I don't think Team is inappropriate variable name. It should be updated to clarify what it's actually doing or what area of of the application it's actually impacting